app/queries/display_ads/filtered_ads_query.rb

```diff
@@ -4,10 +4,11 @@ def self.call(...)
       new(...).call
     end
 
-    def initialize(display_ads:, area:, user_signed_in:, article_tags: [])
+    def initialize(display_ads:, area:, user_signed_in:, organization_id:, article_tags: [])
```

Comment:
Since it defaults to `nil` in `DisplayAd.for_display`, we could probably also have it be optional here as well:

```suggestion
    def initialize(display_ads:, area:, user_signed_in:, organization_id: nil, article_tags: [])
```

(would help with some of the test failures, I think)